Mlem/App/Views/Root/ContentView.swift

```diff
@@ -131,4 +145,34 @@ struct ContentView: View {
             )
         }
     }
+    
+    func loadAvatar(url: URL) async {
+        do {
+            let imageTask = ImagePipeline.shared.imageTask(with: url.withIconSize(128))
+            var avatarImage: UIImage? = try await imageTask.image
+            
+            if let image = avatarImage {
+                avatarImage = image
+                    .resized(to: .init(width: image.size.width / image.size.height * 26, height: 26))
+                    .circleMasked
+                    .withRenderingMode(.alwaysOriginal)
+            }
+            
+            var selectedAvatarImage: UIImage? = try await imageTask.image
+            
+            if let image = avatarImage {
+                selectedAvatarImage = image
+                    .resized(to: .init(width: image.size.width / image.size.height * 26, height: 26))
+                    .circleBorder(color: .init(palette.accent), width: 3.5)
+                    .withRenderingMode(.alwaysOriginal)
+            }
+            
+            Task { @MainActor in
+                self.avatarImage = avatarImage
+                self.selectedAvatarImage = selectedAvatarImage
+            }
```

Comment:
This block causes build failures in Xcode 15:

`Reference to captured var 'avatarImage' in concurrently-executing code`
`Reference to captured var 'selectedAvatarImage' in concurrently-executing code`

Reply:
Fixed 👍